modules/openapi-generator/src/main/resources/python/python-experimental/api.mustache

```diff
@@ -216,7 +216,7 @@ class {{classname}}(object):
                 },
                 'openapi_types': {
 {{#allParams}}
-                    '{{paramName}}': ({{{dataType}}},),
+                    '{{paramName}}': ({{{dataType}}},),   # noqa: E501
```

Comment:
Can we spread this assignment over a newline and eliminate the `# noqa: E501`?

Reply:
ok... That's what I did initially, but then I noticed developers seem to be using # noqa: E501

Reply:
making change. BTW, in the future it may not be very obvious why the line is split in two lines.

Reply:
True, though if they change it they will get the sad E501 errors.
Thank you for your understanding.